mbed_app.json

```diff
@@ -244,6 +244,44 @@
             "lwip.ethernet-enabled"                     : false,
             "lwip.mem-size"                             : 22000,
             "nsapi.dns-response-wait-time"              : 30000
+        },
+        "RZ_A1H": {
+            "target.extra_labels_add"                   : ["PSA"],
+            "target.macros_add"                         : ["MBEDTLS_USER_CONFIG_FILE=\"mbedTLSConfig_mbedOS_SW_TRNG_PSA.h\"", "MBEDTLS_PSA_CRYPTO_C", "MBEDTLS_ENTROPY_NV_SEED", "PAL_USE_HW_TRNG=0"],
+            "target.bootloader_supported"               : true,
+            "target.network-default-interface-type"     : "ETHERNET",
+            "target.bootloader_img"                     : "bootloader/mbed-bootloader-gr_peach-internal_flash-no_rot-v4.1.2.bin",
+            "target.header_offset"                      : "0x10000",
+            "target.app_offset"                         : "0x10400",
+            "bootloader-size"                           : "(64*1024)",
+            "update-client.bootloader-details"          : "0x1800a660",
+            "update-client.application-details"         : "(MBED_ROM_START + MBED_BOOTLOADER_SIZE)",
+            "update-client.storage-address"             : "(MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_BASE_ADDRESS+MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_SIZE)",
+            "update-client.storage-size"                : "(4*1024*1024-MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_SIZE)",
+            "update-client.storage-page"                : 1,
+            "mbed-cloud-client.update-storage"          : "ARM_UCP_FLASHIAP",
+            "storage_tdb_internal.internal_base_address": "(MBED_ROM_START+4*1024*1024)",
```

Comment:
Does this board really have 8 MB of internal flash? Is it working in dual-bank mode? This configuration would suggest that it is. Mbed OS documentation seems to say so, but Renessas own documentation does not make very clear mention at all.

Reply:
External flash is not as secure as internal flash, so probably some warning caveats should be put in place. Do the chips have any OTP key functionality?

Reply:
README.md needs to be thus updated with some proper boiler plate (and added to the list of supported devices).

Reply:
I agreed to update the README.md to add some warning about the external flash.